modules/travel_pay/app/controllers/travel_pay/application_controller.rb

```diff
@@ -30,6 +31,17 @@ class ApplicationController < ::ApplicationController
 
     protected
 
+    def scrub_logs
+      logger.filter = -> log do
+        if (log.name =~ /TravelPay/) && (log.payload[:action].eql? 'show')
```

Comment:
For now, I think this is fine, but I do wonder if we need to think about making this more generic to cover other situations where we might need to mask PII.

Reply:
Yeah I have some ideas on that. I added it to the tech debt spreadsheet